frontend/src/pages/admin/Settings/General.vue

```diff
@@ -245,6 +231,15 @@ export default {
                 label: pt.name
             }
         })
+        const teamTypes = await teamTypesApi.getTeamTypes()
+        this.teamTypes = teamTypes.types.map(tt => {
+            return {
+                order: tt.order,
+                value: tt.id,
+                label: tt.name
+            }
+        })
+        this.teamTypes.sort((A, B) => { return A.order - B.order })
```

Comment:
Usually this would go in a `computed` function, but we're not mutating `teamTypes` at any point, so this is ok for now.